config.json

```diff
@@ -1422,6 +1422,17 @@
         ],
         "prerequisites": [],
         "difficulty": 3
+      },
+      {
+        "slug": "dnd-character",
+        "name": "D&D Character",
+        "uuid": "3ad6da95-f3ff-401b-83e7-6b8b7b29e133",
+        "practices": [
+          "classes",
+          "numbers"
+        ],
+        "prerequisites": [],
+        "difficulty": 1
```

Comment:
This is an easy exercise, but I think the difficulty is probably a 2 or 3 based on the ratings we've given other exercises on this track.

Reply:
Bumped to 2. I looked at a few other exercises to get an idea of where it sat, but my sample size may have been smaller and my perception of difficulty different. E.g. I saw that Roman Numerals was ranked 1 and assessed this exercise seemed to me to be conceptually easier than that at first blush?

Anyway, the JS track also ranks it as 2, so that seems like a good call between 2 and 3?

Reply:
Works for me!